src/core/network/workspaceutils.cpp

```diff
@@ -36,26 +36,34 @@
 #include <inviwo/core/util/filesystem.h>
 #include <inviwo/core/util/logerrorcounter.h>
 
+#include <fmt/format.h>
+
 namespace inviwo {
 
 namespace util {
 
-void updateWorkspaces(InviwoApplication* app) {
-    updateWorkspaces(app, filesystem::getPath(PathType::Workspaces));
-
-    for (const auto& m : app->getModules()) {
-        updateWorkspaces(app, m->getPath(ModulePath::Workspaces));
-    }
-}
+void forEachWorkspaceInDirRecusive(std::string_view path,
+                                   std::function<void(std::string_view)> callback) {
+    std::function<void(const std::string&)> traverse = [&](const std::string& path) {
```

Comment:
why not using `filesystem::getDirectoryContentsRecursively()` here?

Reply:
guess so... the function is old I just moved it